### PR TITLE
reader: morph lines to 94 characters if they end in spaces

### PR DESCRIPTION
**Pull request by adamdecaf**

### CODE REVIEW

File: reader.go [hunk 1]
```diff
@@ -118,8 +118,20 @@ func (r *Reader) Read() (File, error) {
 			if err := r.processFixedWidthFile(&line); err != nil {
 				r.errors.Add(err)
 			}
+
 		case lineLength != RecordLength:
-			r.errors.Add(r.parseError(NewRecordWrongLengthErr(lineLength)))
+			if lineLength > RecordLength {
+				// take the first 94 characters
```
Comment (wadearnold) [inline reader.go]: Should we only take these if all the characters are blank? We should fail if data exists beyond RecordLength

Comment (wadearnold) [reply reader.go]: I have never seen the use case that I am describing. Maybe I am overthinking this and we just got with this until we see a use case that requires the additional check.

**Review by wadearnold — APPROVED** [review]
This is better than what was previously there regardless of my comments.

File: reader.go (hunk 1 above)
Comment (adamdecaf) [reply reader.go]: I agree we should only trim extra spaces. Additional non-space characters would likely indicate an error in one of the fields.

**Review by wadearnold — APPROVED** [review]

**Pull request merged**